cli-scaffold.js

```diff
@@ -0,0 +1,216 @@
+const yaml = require('js-yaml'),
+    fs = require("fs");
+const colorRed = '\x1b[31m',
+    colorCyan = '\x1b[36m',
+	colorReset = '\x1b[0m';
+
+module.exports = function(args, cb)
+{
+    if (!checkArgs(args, cb))
+    {
+        return;
+    }
+
+    let cmd = Object.keys(args)[0],
+        waveId = args[cmd];
+    
+    createTest(cmd, waveId, cb);
+}
+
+function createTest(cmd, waveId, cb)
+{
+    // lib/waves/{wave id} folder must be empty
+    let waveFolder = 'lib/waves/' + waveId,
+        folderExist = fs.existsSync(waveFolder);
+    if (folderExist && fs.readdirSync(waveFolder).length)
+    {
+        setTimeout(function(){
+            console.warn(`${colorRed}%s${colorReset}`, `Wave id [${waveId}] already exists.`);
+        });
+        cb();
+        return;
+    }
+
+    if (!folderExist)
+    {
+        fs.mkdirSync(waveFolder, {recursive: true});
+    }
+
+    if (cmd == 'new')
```

Comment:
Should we use strict mode equality here for consistency with the modular build script?

Reply:
'use strict'; added.